worlds/ffmq/Items.py

```diff
@@ -286,7 +308,10 @@ class FFMQItem(Item):
     type = None
 
     def __init__(self, name, player: int = None):
-        item_data = item_table[name]
+        if name in item_table:
+            item_data = item_table[name]
+        else:
+            item_data = ItemData(None, ItemClassification.progression)
```

Comment:
```suggestion
        item_data = item_table.get(name, ItemData(None, ItemClassification.progression))
```
Seems like you could use `.get`.